STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.Headers.cs

```diff
@@ -0,0 +1,1778 @@
+
+            HttpExchangeContentHeaders httpExchangeContentHeaders = new HttpExchangeContentHeaders
+            {
+                ContentDisposition = invalidContentDispositionHeader
+            };
+
+            return new
+            {
+                HttpExchangeContentHeaders = httpExchangeContentHeaders,
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateContentEncodingHeaderException(string[] invalidContentEncodingHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange content header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.ContentEncoding),
+                value: "Content Encoding header has invalid configuration, fix errors and try again.");
+
+            HttpExchangeContentHeaders httpExchangeContentHeaders = new HttpExchangeContentHeaders
+            {
+                ContentEncoding = invalidContentEncodingHeader
+            };
+
+            return new
+            {
+                HttpExchangeContentHeaders = httpExchangeContentHeaders,
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateContentLanguageHeaderException(string[] invalidContentLanguageHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange content header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.ContentLanguage),
+                value: "Content Language header has invalid configuration, fix errors and try again.");
+
+            HttpExchangeContentHeaders httpExchangeContentHeaders = new HttpExchangeContentHeaders
+            {
+                ContentEncoding = invalidContentLanguageHeader
+            };
+
+            return new
+            {
+                HttpExchangeContentHeaders = httpExchangeContentHeaders,
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateContentRangeHeaderException(
+            ContentRangeHeader invalidContentRangeHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange content header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.ContentRange),
+                value: "Content Range Header has invalid configuration, fix errors and try again.");
```

Comment:
"Header" with lower case. Stay consistent to previous messages.

Reply:
Resolved